### ISSUE TITLE
data integrity from joins

### ISSUE THREAD

**Issue by f-hafner** [f-hafner/magutils#7]
- unique links
- no duplicate records within id-year etc

how to implement:
```R
res <- dbSendQuery(
  conn = con,
  statement = "select type, tbl_name, sql from sqlite_master where type = 'index'"
)
dbFetch(res) # returns a df with the relevant information
dbClearResult()

# extract the index columns and whether it is unique
```

**Comment by f-hafner** [comment]
use the same approach for copying indexes from the production to the mockdb

**Comment by f-hafner** [comment]
the first part is implemented in commit caf9561f51b5340d5e6a555a192eaaca4b89e27a. Closing this, see #13 for copying indexes.
